src/credentials/attestation_verifier/DeviceAttestationVerifier.h

```diff
@@ -47,6 +47,7 @@ enum class AttestationVerificationResult : uint16_t
     kPaiVendorIdMismatch  = 205,
     kPaiAuthorityNotFound = 206,
     kPaiMissing           = 207,
+    kPaiAndDacRevoked     = 208,
```

Comment:
Have a different code for PAI revoked, and DAC revoked, in the correct ranges (200 for PAI, 300 for DAC)

Reply:
Those already exist.  This is the code for "they are both revoked".